packages/gatsby-theme-vtex/src/components/Search/SearchTemplate.tsx

```diff
@@ -0,0 +1,44 @@
+/** @jsx jsx */
+import { jsx } from '@vtex/store-ui'
+import { FC } from 'react'
+
+export interface Props {
+  aside: JSX.Element
+}
+
+export const SearchTemplateContainer: FC = ({ children }) => (
+  <div
+    sx={{
+      display: 'flex',
+      flexWrap: 'wrap',
+    }}
+  >
+    {children}
+  </div>
+)
+
+export const SearchTemplateAside: FC = ({ children }) => (
+  <aside
+    sx={{
+      display: ['none', 'block'],
+      flexGrow: 1,
+      flexBasis: 'sidebar',
+      width: 230,
+    }}
+  >
+    {children}
+  </aside>
+)
+
+export const SearchTemplateMain: FC = ({ children }) => (
+  <div
+    sx={{
+      flexGrow: 99999,
+      flexBasis: 0,
+      minWidth: 300,
+      ml: [0, '3rem'],
+    }}
```

Comment:
should the styles in this file be applied using `sx`?

Reply:
I think so. This file is mainly page structure. If we feel the need we can add a variant based approach in the future